lib/config.js

```diff
@@ -8,5 +8,6 @@ export default {
     channel: '#' + process.env.SLACK_CHANNEL, // Slack channel without # prefix
     username: process.env.SLACK_USERNAME || 'MyAwesomeBot',
     githubUser: process.env.GITHUB_USER,
-    githubToken: process.env.GITHUB_ACCESS_TOKEN
+    githubToken: process.env.GITHUB_ACCESS_TOKEN,
+    extMode: process.env.EXT_MODE
```

Comment:
What about `if there is a google sheet config file present / configured we auto-switch to sheet mode, otherwise fallback to plain json file`? This way we make it easier to configure (not having to). What do you think?